src/board/audio.ts

```diff
@@ -18,8 +21,43 @@ export class AudioUI {
       // Match the regular audio rate.
       sampleRate: 7812 * 4,
     });
-    this.default = new BufferedAudio(this.context, defaultAudioCallback);
-    this.speech = new BufferedAudio(this.context, speechAudioCallback);
+    this.programVolume = 128;
+    this.gainNode = this.context.createGain();
+    this.setVolume(this.programVolume, true)
+    this.default = new BufferedAudio(this.context, this.gainNode, defaultAudioCallback);
+    this.speech = new BufferedAudio(this.context,this.gainNode, speechAudioCallback);
```

Comment:
```suggestion
    this.speech = new BufferedAudio(this.context,this.gainNode, speechAudioCallback);
```

Missing space between the commas. Are these files formatting via Prettier on save for you?

Reply:
Ahh, they are not.